common/dev/adc.c

```diff
@@ -74,19 +74,30 @@ bool adc_init()
 	return true;
 }
 
+int bat_3v_set_gpio(uint8_t sensor_num, void *arg)
+{
+	SET_GPIO_VALUE_CFG *cfg = arg;
```

Comment:
Check to make sure arg != NULL

Reply:
The optimization code is based on v2022.07.01. After that, the main branch has been phased in some
commits related to sensor monitoring. Those changes will be phased in main_v2 in the other PRs. This
problem will be solved in those PRs.
Please check #223 .

Reply:
The optimization code is based on v2022.07.01. After that, the main branch has been phased in some
commits related to sensor monitoring. Those changes will be phased in main_v2 in the other PRs. This
problem will be solved in those PRs.
Please check #223  .